webapp-django/crashstats/settings/base.py

```diff
@@ -682,3 +682,11 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 
 CRASH_ANALYSIS_URL = 'https://crash-analysis.mozilla.com/crash_analysis/'
+
+# At what point do we consider crontabber to be stale.
+# Ie. if it hasn't run for a certain number of hours we'd consider
+# that a failing situation and it'll trigger monitoring.
+CRONTABBER_STALE_HOURS = config(
+    'CRONTABBER_STALE_HOURS',
+    default=24
```

Comment:
Yeah, I'm not so sure myself. We do run a lot of jobs around one area of the day which we consider to be the night. However, we know we have some jobs that are supposed to run every hour. And there's even a job that's supposed to run every 5 min!

However, our goal is to reduce our dependency on cron jobs in general and go towards more run-time reports. I.e. less matviews and more supersearch. 

Heck, to make sense of it all, let's change the default down to 1 hour.